### ISSUE TITLE
[ENH] Support all CLI options in .nodegenerc

### ISSUE THREAD

**Issue by p-mcgowan** [acrontum/generate-it#67]
What do you think?

instead of long lines and verbose package scripts, would be nice to just say `generate:nodegen": "generate-it"`.

Then nodegen could just contain
```
{
  "nodegenDir": "src/http",
  "helpers": {
    "stub": {
      "requestType": "NodegenRequest"
    }
  },
  "confirm": true, // --yes
  "mocked": true,
  "templates": "github.com/p-mcgowan/burn-it-down",
  "inputSpec": "../swagger/build/api.yml"
}
```

**Comment by Acrontum-Carmichael** [comment]
yes - 1 point though.. how to get the yaml path in there to start with?

**Comment by p-mcgowan** [comment]
I think the auto generated nodegenerc could stay the same (with recommended settings), but if it allows for these settings then we can manually update package.json if we want to trim some stuff.

Even `generate-it --template "github/tpls" ../swagger/build/api.yml` would be much cleaner - either way, you need to know those ahead of time to do the first gen. 

So `npm init --yes && npm i -D generate-it && <add gen script> && npm run gen && <manually change the package and nodegen scripts>`

At least this way you can avoid a 300 char package script, and this is a fairly common pattern with stuff like webpack or typesescript or frameworks
